doc/index.md

```diff
@@ -41,6 +41,10 @@ Preferences
 
         <preference name="StatusBarBackgroundColor" value="#000000" />
 
+- __AndroidStatusBarBackgroundColor__ (color hex string, defaults to the Android theme default). On Android 5 and up, the background color can be set by a hex string (#RRGGBB) at startup. We don't use the same property as for iOS because on iOS you typically want the statusbar to have the same color as the app background, but the Android 5+ guidelines specify using a different color than you apps main color, so the value of this property is typically different than the one specified by StatusBarBackgroundColor.
```

Comment:
@EddyVerbruggen Like this?
<preference name="Fullscreen" value="false" />
<preference name="Orientation" value="portrait" />
<platform name="ios">
    <preference name="BackupWebStorage" value="none" />
    <preference name="UIWebViewDecelerationSpeed" value="fast" />
    <preference name="StatusBarOverlaysWebView" value="false" />
    <preference name="StatusBarStyle" value="lightcontent" />
    <preference name="StatusBarBackgroundColor" value="#000000" />
</platform>
<platform name="android">
    <preference name="StatusBarBackgroundColor" value="#00FF00" />
</platform>